node_lambnik/env-template

```diff
@@ -10,4 +10,16 @@ PROD_TILEGARDEN_PASSWORD=
 PROD_TILEGARDEN_USER=
 
 # Function config information
-TIMEOUT=5
+# REQUIRED
+LAMBDA_REGION=us-east-1
+LAMBDA_ROLE=role
+
+# OPTIONAL
+# Amount of time in seconds your lambdas will wait before timing out
+#LAMBDA_TIMEOUT=5
+# Memory in MB allocated to your lambda functions
+#LAMBDA_MEMORY=3008
+# VPC Subnets that your lambdas should use (comma separated list)
+#LAMBDA_SUBNETS=subnet1,subnet2,subnet...N
+# VPC Security Groups that your lambdas should use (comma separated list)
+#LAMBDA_SECURITY_GROUPS=group1,group2,group...N
```

Comment:
I don't think it's totally clear what the `SUBNETS` and `SECURITY_GROUPS` variables are for and how to find values for them.  It might be more detail than we want in the README, so maybe a bit more info in the comments in this file would be the way to go.  E.g. "If you're using RDS, the subnets and security groups should match those of your RDS instance."